v2.0/inverted-indexes.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Inverted Indexes
+summary: Inverted indexes improve your database's performance and usefulness by helping SQL locate schemaless data.
+toc: false
+toc_not_nested: true
+---
+
+<span class="version-tag">New in v2.0:</span> Inverted indexes improve your database's performance by helping SQL locate schemaless data (i.e., [`JSONB`](jsonb.html)).
+
+<div id="toc"></div>
+
+## How Do Inverted Indexes Work?
+
+Standard [indexes](indexes.html) work well for searches based on prefixes of sorted data. However, schemaless data like `JSONB` cannot be queried without a full table scan, since it does not adhere to ordinary value prefix comparison operators. `JSONB` needs to be indexed in a more detailed way than what a standard index provides. This is where inverted indexes prove useful.
```

Comment:
Can you make the `JSONB` instance links to the data type page?

Reply:
Are you suggesting that all `JSONB` mentions in this doc be links to jsonb.html?

Reply:
Not sure about all, but at least 1. Your call.